frontend/src/modules/automation/components/automation-form.vue

```diff
@@ -287,6 +291,14 @@ const doSubmit = () => {
   }
 };
 
+onMounted(() => {
+  fetchIntegrations();
+  storeUnsubscribe.value = store.subscribeAction((action) => {
+    if (action.type === 'auth/doRefreshCurrentUser') {
+      fetchIntegrations();
+    }
+  });
+});
```

Comment:
Do we really need to fetch integrations on user refresh? I'm just asking because I'm not seeing a scenario where the user is updated when in the automations form. It seems that it should be enough to do the first `fetchIntegratoins` onMounted

But please correct me if I'm wrong 😄 
Nonetheless, if we really need this then I don't think that `storeUnsubscribe` is defined anywhere and we should unsubscribe to the action on unmount